Nighttime-Operations/Auxiliary-Telescope/AT-On-Sky/Focus-center-absorbPointingOffsets.rst

```diff
@@ -0,0 +1,102 @@
+.. |author| replace:: *TDB*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _AT-On-sky-CWFS:
+
+##################################################################
+Beginning of the night: Focus, center and absorb pointing offsets
+##################################################################
+
+.. _Beginning_of_the_night-the-Telescope-Overview:
+
+Overview
+========
+
+This procedure must be followed by the observers before the beginning of dark time, prior to starting science or engineering activities.
+It will focus the telescope, center a target and absorb the pointing offsets. 
+
+.. _Beginning_of_the_night-Prerequisites:
+
+Prerequisites
+=============
+- The telescope is ready for on-sky operations: it has been vented (if applicable), mirror cover and dome shutter are open, AOS corrections are enabled. (:ref:`Prepare for on-sky <AT-Pre-Open-for-On-Sky-Operations>`)
+- The sky brightness is low enough that a CWFS can converge succesfully. 
+  This condition is normally achieved around 10-15 minutes before the end of nautical twilight or dark time. 
+
+.. _Beginning_of_the_night-Post-Conditions:
+
+Post-Condition
+===============
+
+- Telescope is focused and pointing offsets have been absorbed. 
+  The system is ready to start scheduler-driven observations (:ref:`Scheduler <Nighttime-Scheduler-Scheduler-Operational-Procedures>`) or engineering tasks. 
+
+.. _Beginning_of_the_night-Procedure-Steps:
+
+Procedure Steps
+===============
+
+- Confirm sky background is dark enough. 
+
+As a general rule, a good time to start executing the following SAL scripts is 10-15 minutes before the end of nautical twilight. 
+
+- Launch a CWFS sequence on an 8th magnitude star. 
+
+From the LOVE ``ATQueue`` panel, under ``AVAILABLE SCRIPTS`` on the left side, find and load the script ``auxtel/latiss_cwfs_align.py`` by clicking on the blue icon. 
+It is included under ``External scripts``. 
+
+In the ``configuring Script: latiss_cwfs_align`` window, the script configuration is introduced under ``CONFIG``.
+Traditionally, we perform the first CWFS of the night towards the south and at mid-elevations on an 8-mag star. 
+With the configuration below, the script will query SIMBAD for a target around the provided ``az`` and ``el`` coordinates and with the specified maximum brightness ``mag_limit``.
+
+.. code-block:: text
```

Comment:
I think you can use "yaml" instead of "text"

Reply:
👍